### PR TITLE
Removes bloat comments

### PR DESCRIPTION
**Pull request by Zth--**
I made a script to detect duplicated procs and I have plans of adding it to travis. It's not obvious when people unintentionally redefine a proc because they of course don't appear on code reviews and byond doesn't complain. As far as I know this is a feature we're not using and it has caused a few bugs already.

On the first run of this script I encountered 170 duplicated procs, many of which were comments.
 Unmaintainable, incorrect and unnecesary comments. This PR addresses only these comments (and one duplicated file).

[system]

### CODE REVIEW

**Comment by Zth--** [comment]
Here's the list of duplicated procs after this PR if someone's interested (I haven't polished the script yet so there may be a few false positives)

```
/obj/structure/spacepod_frame/attackby()
	code/WorkInProgress/pomf/spacepods/construction.dm:L27
	code/WorkInProgress/pomf/spacepods/construction.dm:L45
/obj/machinery/computer/rust_gyrotron_controller/Topic()
	code/WorkInProgress/Cael_Aislinn/Rust/gyrotron_controller.dm:L23
	code/WorkInProgress/Cael_Aislinn/Rust/gyrotron_controller.dm:L95
/obj/machinery/rust_fuel_compressor/attackby()
	code/WorkInProgress/Cael_Aislinn/Rust/fuel_compressor.dm:L24
	code/WorkInProgress/Cael_Aislinn/Rust/fuel_compressor_construction.dm:L42
/obj/machinery/rust_fuel_assembly_port/attackby()
	code/WorkInProgress/Cael_Aislinn/Rust/fuel_assembly_port_construction.dm:L41
	code/WorkInProgress/Cael_Aislinn/Rust/fuel_assembly_port.dm:L16
/atom/movable/to_bump()
	code/ZAS/Airflow.dm:L212
	code/game/atoms_movable.dm:L387
/obj/item/mecha_parts/mecha_equipment/tool/extinguisher/can_attach()
	code/game/mecha/equipment/tools/tools.dm:L325
	code/game/mecha/equipment/tools/tools.dm:L395
/obj/item/mecha_parts/mecha_equipment/jetpack/attach()
	code/game/mecha/equipment/tools/tools.dm:L427
	code/game/mecha/equipment/tools/tools.dm:L440
/obj/item/weapon/spellbook/oneuse/ancient/recoil()
	code/game/gamemodes/wizard/spellbook_oneuse.dm:L505
	code/game/gamemodes/wizard/spellbook_oneuse.dm:L518
/obj/machinery/teleport/station/attack_ai()
	code/game/machinery/teleporter.dm:L65
	code/game/machinery/teleporter.dm:L335
/obj/machinery/sleeper/process()
	code/game/machinery/Sleeper.dm:L243
	code/game/machinery/Sleeper.dm:L470
/obj/machinery/newscaster/attack_ai()
	code/game/machinery/newscaster.dm:L211
	code/game/machinery/newscaster.dm:L948
/obj/machinery/telecomms/unlinkFrom()
	code/game/machinery/telecomms/machine_interactions.dm:L291
	code/game/machinery/telecomms/telecomunications.dm:L173
/obj/structure/window/reinforced/holo/spawnBrokenPieces()
	code/game/machinery/computer/HolodeckControl.dm:L517
	code/game/machinery/computer/HolodeckControl.dm:L527
/obj/structure/window/holo/spawnBrokenPieces()
	code/game/machinery/computer/HolodeckControl.dm:L530
	code/game/machinery/computer/HolodeckControl.dm:L540
/obj/machinery/computer/med_data/attack_paw()
	code/game/machinery/computer/Operating.dm:L45
	code/game/machinery/computer/medical.dm:L24
/obj/machinery/door/airlock/bump_open()
	code/game/machinery/doors/airlock.dm:L369
	code/game/machinery/doors/airlock.dm:L396
/obj/machinery/door/Bumped()
	code/game/machinery/doors/airlock.dm:L388
	code/game/machinery/doors/door.dm:L80
/obj/item/toy/balloon/inflated/attackby()
	code/game/objects/items/toys.dm:L1221
	code/game/objects/items/toys.dm:L1390
/obj/item/candle/attackby()
	code/game/objects/items/candle.dm:L25
	code/game/objects/items/candle.dm:L113
/obj/item/weapon/match/is_hot()
	code/game/objects/items/candle.dm:L71
	code/game/objects/items/weapons/cigs_lighters.dm:L115
/obj/item/weapon/dnainjector/nofail/antiblind/initialize()
	code/game/objects/items/weapons/dna_injector.dm:L864
	code/game/objects/items/weapons/dna_injector.dm:L886
/obj/item/weapon/paper/attack()
	code/game/objects/items/weapons/cosmetics.dm:L76
	code/game/objects/items/weapons/cosmetics.dm:L151
/obj/item/weapon/storage/box/lights/tubes/New()
	code/game/objects/items/weapons/storage/boxes.dm:L756
	code/game/objects/items/weapons/storage/boxes.dm:L772
/obj/item/weapon/implant/chem/New()
	code/game/objects/items/weapons/implants/implant.dm:L229
	code/game/objects/items/weapons/implants/implant.dm:L271
/datum/pda_app/station_map/Destroy()
	code/game/objects/items/devices/PDA/apps.dm:L126
	code/game/objects/items/devices/PDA/apps.dm:L142
/obj/item/device/pda/New()
	code/game/objects/items/devices/PDA/PDA.dm:L193
	code/game/objects/items/devices/PDA/PDA.dm:L617
/obj/effect/alien/weeds/node/New()
	code/game/objects/effects/aliens.dm:L207
	code/game/objects/effects/aliens.dm:L244
/obj/effect/effect/smoke/Destroy()
	code/game/objects/effects/effect_system.dm:L260
	code/game/objects/effects/effect_system.dm:L408
/obj/effect/decal/cleanable/Destroy()
	code/game/objects/effects/decals/cleanable.dm:L90
	code/game/objects/effects/decals/cleanable.dm:L113
/obj/structure/falsewall/update_icon()
	code/game/objects/structures/false_walls.dm:L172
	code/game/objects/structures/false_walls.dm:L226
/obj/structure/table/attack_hand()
	code/game/objects/structures/tables_racks.dm:L290
	code/game/objects/structures/tables_racks.dm:L789
/obj/structure/bed/chair/vehicle/firebird/to_bump()
	code/game/objects/structures/vehicles/firebird.dm:L64
	code/game/objects/structures/vehicles/firebird.dm:L71
/obj/structure/closet/Cross()
	code/game/objects/structures/crates_lockers/crates.dm:L425
	code/game/objects/structures/crates_lockers/closets.dm:L84
/obj/structure/closet/statue/take_contents()
	code/game/objects/structures/crates_lockers/closets/statue.dm:L139
	code/game/objects/structures/crates_lockers/closets/statue.dm:L145
/obj/structure/closet/statue/open()
	code/game/objects/structures/crates_lockers/closets/statue.dm:L142
	code/game/objects/structures/crates_lockers/closets/statue.dm:L148
/turf/bullet_act()
	code/game/turfs/turf.dm:L95
	code/game/turfs/turf.dm:L101
/turf/unsimulated/wall/rock/canSmoothWith()
	code/game/turfs/unsimulated/walls.dm:L34
	code/game/turfs/unsimulated/walls.dm:L41
/turf/simulated/wall/Destroy()
	code/game/smoothwall.dm:L97
	code/game/turfs/simulated/walls.dm:L370
/datum/objective/summon_narsie/feedbackText()
	code/datums/gamemode/objectives/spray_blood.dm:L21
	code/datums/gamemode/objectives/summon_narsie.dm:L9
/datum/objective/summon_narsie/feedbackText()
	code/datums/gamemode/objectives/spray_blood.dm:L21
	code/datums/gamemode/objectives/convert_people.dm:L38
/obj/item/weapon/storage/backpack/cultpack/cultify()
	code/game/objects/items/weapons/storage/backpack.dm:L52
	code/datums/gamemode/factions/bloodcult/bloodcult_items.dm:L1076
/datum/rune_spell/blood_cult/summoncultist/abort()
	code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm:L1975
	code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm:L2049
/datum/symptom/weight_loss/Activate()
	code/datums/diseases/advance/symptoms/weight.dm:L68
	code/datums/diseases/advance/symptoms/weight.dm:L110
/datum/command_alert/power_restored/announce()
	code/datums/helper_datums/command_alerts.dm:L285
	code/datums/helper_datums/command_alerts.dm:L423
/obj/machinery/atmospherics/unary/thermal_plate/hide()
	code/ATMOSPHERICS/components/unary/thermal_plate.dm:L57
	code/ATMOSPHERICS/components/unary/thermal_plate.dm:L89
/proc/notes_add()
	code/modules/admin/player_notes.dm:L35
	code/modules/admin/player_notes.dm:L95
/datum/disease2/effect/deaf/activate()
	code/modules/virus2/effect/stage_4.dm:L75
	code/modules/virus2/effect/stage_3.dm:L68
/obj/machinery/portable_atmospherics/hydroponics/proc/process_reagents()
	code/modules/hydroponics/hydroponics_reagents.dm:L24
	code/modules/hydroponics/hydroponics_reagents.dm:L355
/obj/machinery/disposal/Destroy()
	code/modules/recycling/disposal.dm:L64
	code/modules/recycling/disposal.dm:L539
/obj/machinery/light/New()
	code/modules/power/lighting.dm:L128
	code/modules/power/lighting.dm:L204
/obj/machinery/singularity/deadchat_controlled/Destroy()
	code/modules/power/singularity/singularity.dm:L702
	code/modules/power/singularity/singularity.dm:L725
/obj/structure/particle_accelerator/update_icon()
	code/modules/power/singularity/particle_accelerator/particle_accelerator.dm:L86
	code/modules/power/singularity/particle_accelerator/particle_accelerator.dm:L164
/obj/item/weapon/reagent_containers/attackby()
	code/modules/reagents/reagent_containers.dm:L136
	code/modules/reagents/reagent_containers.dm:L376
/obj/item/weapon/reagent_containers/glass/bottle/New()
	code/modules/reagents/reagent_containers/glass/bottle.dm:L19
	code/modules/reagents/reagent_containers/glass/bottle.dm:L40
/obj/item/weapon/reagent_containers/food/drinks/soda_cans/attackby()
	code/game/objects/items/weapons/grenades/ghettobomb.dm:L4
	code/modules/reagents/reagent_containers/food/drinks.dm:L632
/obj/item/weapon/reagent_containers/food/condiment/attackby()
	code/modules/reagents/reagent_containers/food/condiment.dm:L20
	code/modules/reagents/reagent_containers/food/condiment.dm:L78
/obj/item/weapon/reagent_containers/food/snacks/chicken_nuggets/New()
	code/modules/reagents/reagent_containers/food/snacks.dm:L3859
	code/modules/reagents/reagent_containers/food/snacks.dm:L3881
/obj/item/weapon/reagent_containers/food/snacks/grown/New()
	code/modules/reagents/reagent_containers/food/snacks/grown.dm:L19
	code/modules/reagents/reagent_containers/food/snacks/grown.dm:L33
/obj/item/weapon/reagent_containers/food/snacks/meat/New()
	code/modules/reagents/reagent_containers/food/snacks/meat.dm:L14
	code/modules/reagents/reagent_containers/food/snacks/meat.dm:L19
/atom/New()
	code/game/atoms.dm:L190
	code/modules/awaymissions/maploader/reader.dm:L410
/obj/item/weapon/reagent_containers/glass/beaker/water/New()
	code/modules/research/xenoarchaeology/chemistry.dm:L148
	code/modules/research/xenoarchaeology/chemistry.dm:L157
/mob/proc/emote()
	code/modules/mob/say.dm:L91
	code/modules/mob/emote.dm:L2
/mob/living/carbon/brain/say()
	code/modules/mob/living/carbon/brain/brain.dm:L35
	code/modules/mob/living/carbon/brain/say.dm:L1
/mob/living/carbon/alien/humanoid/Login()
	code/modules/mob/living/carbon/alien/login.dm:L1
	code/modules/mob/living/carbon/alien/humanoid/login.dm:L1
/mob/living/carbon/human/tajaran/New()
	code/modules/mob/living/carbon/human/human.dm:L30
	code/modules/mob/living/carbon/human/catbeast/catbeast.dm:L1
/client/MouseDrop()
	code/WorkInProgress/buildmode.dm:L234
	code/modules/mob/living/simple_animal/shade_powers.dm:L250
/mob/living/simple_animal/shade/Life()
	code/modules/mob/living/simple_animal/shade.dm:L72
	code/modules/mob/living/simple_animal/shade.dm:L131
/mob/living/simple_animal/vox/armalis/attackby()
	code/modules/mob/living/simple_animal/vox.dm:L42
	code/modules/mob/living/simple_animal/vox.dm:L122
/mob/living/simple_animal/hologram/RangedAttack()
	code/modules/mob/living/simple_animal/hologram/hologram.dm:L42
	code/modules/mob/living/simple_animal/hologram/hologram.dm:L108
/mob/living/simple_animal/mouse/Move()
	code/modules/mob/living/simple_animal/friendly/mouse.dm:L219
	code/modules/mob/living/simple_animal/friendly/mouse.dm:L359
/mob/living/simple_animal/hostile/retaliate/cluwne/attack_alien()
	code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm:L192
	code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm:L231
/mob/living/simple_animal/hostile/retaliate/cluwne/attackby()
	code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm:L182
	code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm:L256
/mob/living/carbon/can_use_hands()
	code/modules/mob/living/carbon/carbon.dm:L341
	code/modules/mob/living/silicon/mommi/inventory.dm:L334
/mob/living/silicon/robot/mommi/cogspider/updatename()
	code/modules/mob/living/silicon/mommi/cogspider.dm:L10
	code/modules/mob/living/silicon/mommi/mommi_subtypes/cogspider.dm:L10
/mob/living/silicon/robot/mommi/cogspider/identification_string()
	code/modules/mob/living/silicon/mommi/cogspider.dm:L13
	code/modules/mob/living/silicon/mommi/mommi_subtypes/cogspider.dm:L14
/mob/living/silicon/robot/mommi/cogspider/New()
	code/modules/mob/living/silicon/mommi/cogspider.dm:L17
	code/modules/mob/living/silicon/mommi/mommi_subtypes/cogspider.dm:L17
/mob/living/silicon/robot/identification_string()
	code/modules/mob/living/silicon/mommi/mommi_subtypes/soviet.dm:L14
	code/modules/mob/living/silicon/robot/robot.dm:L1319
/mob/living/silicon/robot/get_inactive_hand()
	code/modules/mob/living/silicon/robot/inventory.dm:L8
	code/modules/mob/living/silicon/robot/inventory.dm:L325
/turf/simulated/New()
	code/game/turfs/simulated.dm:L12
	code/modules/mob/living/silicon/ai/freelook/update_triggers.dm:L18
/mob/living/silicon/robot/Move()
	code/modules/mob/living/silicon/robot/robot.dm:L1145
	code/modules/mob/living/silicon/ai/freelook/update_triggers.dm:L67
/mob/dead/observer/Login()
	code/modules/mob/dead/observer/login.dm:L1
	code/modules/mob/dead/observer/observer.dm:L1103
/mob/dead/observer/Logout()
	code/modules/mob/dead/observer/observer.dm:L1107
	code/modules/mob/dead/observer/logout.dm:L1
/proc/EquipCustomItems()
	code/modules/customitems/item_spawning.dm:L2
	code/modules/customitems/item_spawning.dm:L127
/area/New()
	code/game/area/areas.dm:L23
	code/modules/lighting/lighting_area.dm:L5
/atom/New()
	code/game/atoms.dm:L190
	code/modules/lighting/lighting_atom.dm:L53
/atom/Destroy()
	code/game/atoms.dm:L158
	code/modules/lighting/lighting_atom.dm:L64
/atom/movable/Destroy()
	code/game/atoms_movable.dm:L72
	code/modules/lighting/lighting_atom.dm:L71
/turf/New()
	code/game/turfs/turf.dm:L82
	code/modules/lighting/lighting_turf.dm:L12
/turf/Entered()
	code/game/turfs/turf.dm:L158
	code/modules/lighting/lighting_turf.dm:L73
/turf/unsimulated/wall/cultify()
	code/game/turfs/unsimulated/walls.dm:L91
	code/modules/bomberman/bomberman.dm:L479
/obj/item/trash/pietin/attackby()
	code/game/objects/items/trash.dm:L134
	code/modules/food/customizables.dm:L102
/obj/item/device/gps/attack_self()
	code/modules/research/xenoarchaeology/tools/tools.dm:L13
	code/modules/telesci/gps.dm:L56
/datum/runtimeError/TypeMismatch/New()
	code/modules/scripting/Errors.dm:L113
	code/modules/scripting/Errors.dm:L119
/obj/item/device/assembly/interact()
	code/modules/assembly/assembly.dm:L90
	code/modules/assembly/assembly.dm:L309
/proc/time_stamp()
	code/__HELPERS/time.dm:L53
	code/__HELPERS/time.dm:L57
95 duplicated procs were found.

**Comment by MadmanMartian** [comment]
For the duplicate proc script, can it give the filenames and lines of the procs?

**Comment by PJB3005** [comment]
I appreciate you keeping `/proc/time_stamp()` intact.

**Comment by Zth--** [comment]
@MadmanMartian what about now? I edited the comment.

@PJB3005 I still haven't decided what to do with it, I find it hilarious and won't remove it 

I improved the script and found 30 more occurrences

**Comment by Kharkov72** [comment]
Someone write down this pr just in case we need one of these again

File: code/datums/gamemode/factions/legacy_cult/trash_machinery.dm [hunk 1]
```diff
@@ -1,93 +0,0 @@
-//Machinery that gets deleted by Nar-Sie and isn't replaced with cult structures
```
Comment (Kurfursten) [inline code/datums/gamemode/factions/legacy_cult/trash_machinery.dm:1]: is this all moved elsewhere?

Comment (Probe1) [reply code/datums/gamemode/factions/legacy_cult/trash_machinery.dm:1]: @Zth-- pls respond

Comment (Zth--) [reply code/datums/gamemode/factions/legacy_cult/trash_machinery.dm:1]: This is a duplicated file. You can find the other one here https://github.com/vgstation-coders/vgstation13/blob/817b0d8b685e195454e5311b0cbe3e8b52d1af36/code/datums/gamemode/factions/bloodcult/bloodcult_cultify_exceptions.dm

**Pull request merged**